README.md

```diff
@@ -129,6 +129,8 @@ React.render(<Table columns={columns} data={data} />, mountNode);
 | fixed | String \| Boolean |  | this column will be fixed when table scroll horizontally: true or 'left' or 'right' |
 | align | String |  | specify how cell content is aligned |
 | ellipsis | Boolean |  | specify whether cell content be ellipsized |
+| colScope | 'col' \| 'colgroup' |  | 	Set scope attribute for a column |
```

Comment:
Seems this prop can be auto generate

Reply:
Do you mean that I should remove the colScope prop I added and write code that will automatically set the scope for the columns? (for example, by checking the colspan property)

Reply:
Yes. Since Table always put thead on the top. It's stable structure : )